cli/tests/unit/unit_test_runner.ts

```diff
@@ -225,6 +227,13 @@ async function masterRunnerMain(
   }
 
   console.log("Unit tests passed");
+
+  if (REGISTERED_UNIT_TESTS.find(({ only }) => only)) {
+    console.error(
+      `\n${colors.red("FAILED")} because the "only" option was used`
+    );
+    Deno.exit(1);
+  }
```

Comment:
Does "deno test" also fail when only is used?

Reply:
Yes, see test.

Reply:
So why is exit(1) necessary here?

Reply:
The unit test runner doesn't use `deno test`'s `exitOnFail` because it needs to print summaries etc. Same reason there is another `Deno.exit(1)` above a bit.